src/ansys/fluent/core/launcher/standalone_launcher.py

```diff
@@ -123,8 +128,15 @@ def __init__(
             default is ``True``. You can stop and start the streaming of the
             Fluent transcript subsequently via the method calls, ``transcript.start()``
             and ``transcript.stop()`` on the session object.
+        show_gui : bool, optional
+            Whether to display the Fluent GUI. The default is ``None``,
+            in which case the GUI is not shown. If ``False`` is
+            not explicitly provided, the GUI will be shown if
```

Comment:
```suggestion
            not explicitly provided, the GUI is shown if
```
Use present tense verb.

Reply:
Done. Thank you.